packages/create-rspack/template-vue-ts/rspack.config.ts

```diff
@@ -28,7 +28,7 @@ export default defineConfig({
 					{
 						loader: "builtin:swc-loader",
 						options: {
-							sourceMap: true,
+							sourceMaps: true,
```

Comment:
How about removing the `sourceMaps: true` as the `swc-loader` reads the `devtool` option by default